instruction-error/src/lib.rs

```diff
@@ -385,7 +347,7 @@ impl fmt::Display for InstructionError {
     }
 }
 
-#[cfg(feature = "std")]
+#[cfg(feature = "num-traits")]
```

Comment:
I looked around to see if there was a standard feature name for num-traits. The highest downloaded crate that has it optional uses `num-traits` https://github.com/starkat99/half-rs, and the second just uses `num` https://github.com/jhpratt/deranged.

All that to say, I'm good with `num-traits`